src/applications/check-in/components/pages/AppointmentDetails/index.jsx

```diff
@@ -312,7 +318,10 @@ const AppointmentDetails = props => {
                   <h2 className="vads-u-font-size--sm">
                     {t('where-to-attend')}
                   </h2>
-                  <div data-testid="appointment-details--facility-value">
+                  <p
```

Comment:
I think we should leave the div here and wrap an additional paragraph tag round the text value and add a paragraph tag to the AddressBlock component. The way this is coded up is creating a div inside a paragraph tag which isn't valid HTML.